### PR TITLE
Initial version for Debian

### PR DESCRIPTION
**Pull request by Thulium-Drake**
With these edits (and the one from #134 ) I was able to succesfully configure Chronyd on my Debian system.

Feedback is appreciated! :-)

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -170,13 +170,14 @@
 - name: Generate ptp4l sysconfig file
   template:
     src: ptp4l.sysconfig.j2
-    dest: /etc/sysconfig/ptp4l
+    dest: "{{ timesync_ntp_sysconfig_path }}"
```
Comment (richm) [inline tasks/main.yml:173]: timesync_ptp4l_sysconfig_path

File: tasks/main.yml [hunk 2]
```diff
@@ -170,13 +170,14 @@
...
     backup: true
     mode: 0644
   notify: restart ptp4l
   when:
     - timesync_mode == 2
     - "'linuxptp' in ansible_facts.packages"
+    - timesync_ntp_sysconfig_path | lenght > 0
```
Comment (richm) [inline tasks/main.yml:180]: `timesync_ptp4l_sysconfig_path | length > 0`

File: tasks/main.yml [hunk 3]
```diff
@@ -189,11 +190,12 @@
     - timesync_mode == 2
     - timesync_mode2_hwts
     - "'linuxptp' in ansible_facts.packages"
+    - timesync_phc2sys_sysconfig_path | lenght > 0
```
Comment (richm) [inline tasks/main.yml:193]: length

Comment (Thulium-Drake) [reply tasks/main.yml]: Oops! Sorry, fixed! :)

File: tasks/main.yml (hunk 1 above)
Comment (Thulium-Drake) [reply tasks/main.yml]: Oops! Sorry, fixed! :)

File: tasks/main.yml (hunk 2 above)
Comment (Thulium-Drake) [reply tasks/main.yml]: Oops! Sorry, fixed! :)

**Comment by richm** [comment]
[citest commit:7cb06fc9a2a0efd5832de60ac67fe04384427e65]

File: tasks/main.yml [hunk 4]
```diff
@@ -123,7 +123,7 @@
 - name: Generate chronyd sysconfig file
   template:
     src: chronyd.sysconfig.j2
-    dest: /etc/sysconfig/chronyd
+    dest: "{{ timesync_chrony_sysconfig_path }}"
```
[the remark this answers is not in the document]
Comment (richm) [reply tasks/main.yml:126]: > Are these `timesync_*_sysconfig_path` variables expected to be set by users? If not, they must be marked as variables internal to the role with the `__` prefix, e.g. `__timesync_chrony_sysconfig_path`.

Unfortunately, the timesync role does not follow this convention - if you look at the existing definitions in the `vars/*.yml` files you will see that none of the variables defined there use the `__` prefix, even though none of them are intended to be set by the user.  So I think it is ok in this case that the proposed `timesync_chrony_sysconfig_path` et. al. do not have the `__` prefix.

**Comment by richm** [comment]
@Thulium-Drake please rebase on top of the latest master branch to fix the ansible-lint issue in `tox / python (3.8)`
the other test failures can be ignored

**Comment by richm** [comment]
[citest commit:15aa13ce24cf085909112f29a8e337809caa5a08]

**Comment by richm** [comment]
[citest commit:192cad3b752afffacede39d168c02fb9b8efce39]

**Review by richm — APPROVED** [review]
lgtm

**Pull request merged**